api/v1beta1/hostedcluster_types.go

```diff
@@ -1636,7 +1643,7 @@ type ClusterVersionStatus struct {
 	// desired is the version that the cluster is reconciling towards.
 	// If the cluster is not yet fully initialized desired will be set
 	// with the information available, which may be an image or a tag.
-	Desired Release `json:"desired"`
+	Desired configv1.Release `json:"desired"`
```

Comment:
Strict superset, property wise, so this should be a non-breaking change for clients.